<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectionDialog</class>
 <widget class="QDialog" name="ProjectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Z Projection</string>
  </property>
  <property name="toolTip">
   <string>Use [Base].Proj.tif if it exists</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>189</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="speedComboBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Slow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Intermediate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Direct</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="depthCheckBox">
      <property name="text">
       <string>Smooth depth estimation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="contrastCheckBox">
      <property name="text">
       <string>Contrast calibration</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="usingExistedCheckBox">
      <property name="text">
       <string>Use existing projection</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
